### ISSUE TITLE
Add a site to which all Share features link to and then redirect to the users instance.

### ISSUE THREAD

**Issue by Arbee4ever** [mastodon/mastodon#25447]
### Pitch

I think an official site which asks the user which their main instance is (save in cookies?), and then redirects them to the post on that instance they defined would be nice. 

### Motivation

This would solve the issue of not being able to interact with a post they opened using a link.

**Comment by cperrin88** [comment]
I agree that this is useful but it is not really in line with the principles of federation. If that one site goes down all instances are affected. Someone has to pay for it and maintain security and we all have to put in our trust that this will happen. It is a singular dependency for all instances.

**Comment by Arbee4ever** [comment]
I mean you could make every mastodon instance have their own, which read the configured instance from some kind of localStorage (idk if something exists which lets u do that over multiple websites)
